pages/common/set.md

```diff
@@ -17,3 +17,7 @@
 - Set various options, e.g. enable `vi` style line editing:
 
 `set -o {{vi}}`
+
+- Set the shell to exit as soon as the first error is encountered. Mostly used in scripts:
```

Comment:
Perhaps this will read better? Just a stylistic choice given existing conventions.
```suggestion
- Set the shell to exit as soon as the first error is encountered (mostly used in scripts):
```

Generally we are wary of documenting things that are mostly used in scripts, but in this case I think it's ok.